Chore: adding NRDOT Section (#20202)
newrelic/docs-website: src/content/docs/opentelemetry/nrdot/nrdot-intro.mdx

---
title: New Relic Distribution of OpenTelemetry (NRDOT)
tags:
  - OpenTelemetry
  - NRDOT
metaDescription: New Relic Supported Distribution of OpenTelemetry
freshnessValidatedDate: 2024-05-14
---

> The New Relic Distribution of OpenTelemetry (NRDOT) is a New Relic-supported distribution of the OpenTelemetry project.

Reviewer: I think this sentence is making too strong a claim right now. Maybe this is the aspirational goal, but not the case right now.

Maybe something like, "New Relic Distribution of OpenTelemetry (NRODT) refers to the collection of New Relic-supported distributions of OpenTelemetry project components.

It provides a set of OpenTelemetry components that are tested and supported by New Relic.

> NRDOT is designed to work seamlessly with New Relic products and services, and it is the recommended way to instrument your applications with OpenTelemetry for use with New Relic.

Reviewer: Similar to the previous comment, this implies that NRDOT alternatives to OpenTelemetry components are available in all domains. Let's rephrase to make it clear that NRDOT is recommended when available.

```suggestion
NRDOT components are designed to work seamlessly with New Relic products and services, and when available, are the recommended way to integrate OpenTelemetry with New Relic.
```

<Callout variant="tip">
Currently, NRDOT includes pre-built distributions of the OpenTelemetry Collector, for language APIs and SDKs see our docs on getting started with [APM monitoring](/docs/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro/).
</Callout>

<DocTiles>

>   <DocTile title='Collector' path="/docs/opentelemetry/nrdot/nrdot-collector/">The New Relic Distribution of OpenTelemetry Collector (NRDOT) is a collection of New Relic-supported pre-built distributions of the OpenTelemetry Collector project.</DocTile>

Reviewer: This DocTile feels a bit lonely. Maybe we hold off on tiles until there are additional entries?

Author: I'm fine with that.
